web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -137,19 +139,69 @@ export const awsRegionMap = {
 };
 
 export type Regions = keyof typeof awsRegionMap;
-export type IntegrationExecuteRequest = {
+
+// RdsEngines are the expected backend string values,
+// used when requesting lists of rds databases of the
+// specified engine.
+export type RdsEngines =
```

Comment:
IMO it should be `RdsEngine`. 
Later in the code you have `engine: RdsEngines;` which reads quite weird.